backend/worker.py

```
@@ -0,0 +1,46 @@
+import time
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+# Reads the URL from your local .env file (which is not pushed to GitHub)
+AMQP_URL = os.getenv("AMQP_URL")
+
+if not AMQP_URL:
+    print("[-] ERROR: Missing AMQP_URL variable in .env file!")
+    exit(1)
```

Ruff (0.15.15)

[warning] Use `sys.exit()` instead of `exit`

Replace `exit` with `sys.exit()`

(PLR1722)